src/amo/components/UserProfileEdit/index.js

```diff
@@ -582,7 +604,10 @@ export class UserProfileEditBase extends React.Component<Props, State> {
                 )}
               </p>
 
-              <UserProfileEditNotifications user={user} />
+              <UserProfileEditNotifications
+                onChange={this.onNotificationChange}
```

Comment:
This doesn't seem to be covered in a test. Should there be a test that verifies that `UserProfileEditNotifications` received the expected `onChange` property?

Reply:
it is covered by "stores updated notifications in state", which makes sure the right behavior is triggered when this prop gets executed.